frontend/src/app/cities-upload/cities-upload.component.html

```diff
@@ -0,0 +1,25 @@
+<h1>Insérer les villes de France</h1>
+
+<ngb-alert type="info" [dismissible]="false" id="info-message">
+  <p>
+    Cet écran permet d'insérer les villes de France dans le système, à partir du fichier CSV téléchargeable
+    <a href="https://www.data.gouv.fr/fr/datasets/base-officielle-des-codes-postaux/">ici</a>.
+  </p>
+  <p>
+    Cette opération ne doit être faite qu'une fois (à moins que les communes changent à l'avenir).
```

Comment:
No. The IDs are just there to have a PK in the table. They're not referenced anwhere. We store the actual postal code and city name in the person. That is less normalized, but at least it allows migrating without any problem.